Source/Core/VideoCommon/TextureCacheEntry.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include <map>
```

Comment:
`<memory>` needs to be included as well.

Reply:
Done, thanks!!